packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -142,10 +139,26 @@ class WorkerPool(SyftObject):
     ]
 
     name: str
-    image: Optional[SyftWorkerImage]
+    image_id: Optional[UID]
     max_count: int
     worker_list: List[LinkedObject]
     created_at: DateTime = DateTime.now()
+    worker_image: Optional[SyftWorkerImage]  # patched for default image
```

Comment:
I added this to make things work with the default pool, since its `APIRegistry.api_for` returns `None`